test/metabase/query_processor_test/nested_queries_test.clj

```diff
@@ -92,10 +92,10 @@
 (deftest breakout-fk-column-test
   (mt/test-drivers (mt/normal-drivers-with-feature :nested-queries :foreign-keys)
     (testing "Test including a breakout of a nested query column that follows an FK"
-      (is (= {:rows [[1 174] [2 474] [3 78] [4 39]]
-              :cols [(qp.test/breakout-col (qp.test/fk-col :checkins :venue_id :venues :price))
-                     (qp.test/aggregate-col :count)]}
-             (qp.test/rows-and-cols
+      (is (=? {:rows [[1 174] [2 474] [3 78] [4 39]]
```

Comment:
Updated a bunch of tests to use `=?` instead of `=` so I didn't have to go add the `:join-alias/original` keys to them. So we won't have to update these tests again in the future if we're recording new bits of info